PAWS/PAWSDelegate.swift

```diff
@@ -92,40 +92,12 @@ class PAWSDelegate: SpeziAppDelegate {
         )
         
         return HealthKit {
-            CollectSample(
-                HKQuantityType.electrocardiogramType(),
-                predicate: sharedPredicate,
-                deliverySetting: .background(saveAnchor: false)
-            )
-            CollectSamples(
-                Set(HKElectrocardiogram.correlatedSymptomTypes),
-                predicate: sharedPredicate,
-                deliverySetting: .background(saveAnchor: true)
-            )
-            CollectSample(
-                HKQuantityType(.heartRate),
-                predicate: sharedPredicate,
-                deliverySetting: .manual(safeAnchor: false)
-            )
-            CollectSample(
-                HKQuantityType(.vo2Max),
-                predicate: sharedPredicate,
-                deliverySetting: .manual(safeAnchor: false)
-            )
-            CollectSample(
-                HKQuantityType(.physicalEffort),
-                predicate: sharedPredicate,
-                deliverySetting: .manual(safeAnchor: false)
-            )
-            CollectSample(
-                HKQuantityType(.stepCount),
-                predicate: sharedPredicate,
-                deliverySetting: .manual(safeAnchor: false)
-            )
-            CollectSample(
-                HKQuantityType(.activeEnergyBurned),
-                predicate: sharedPredicate,
-                deliverySetting: .manual(safeAnchor: false)
+            CollectSample(.electrocardiogram, continueInBackground: true, predicate: sharedPredicate)
+            for correlatedSymptomType in HKElectrocardiogram.correlatedSymptomTypes {
+                CollectSample(correlatedSymptomType, continueInBackground: true, predicate: sharedPredicate)
+            }
+            RequestReadAccess(
+                quantity: [.heartRate, .vo2Max, .physicalEffort, .stepCount, .activeEnergyBurned]
```

Comment:
@lukaskollmer Can you double-check that this is fully equivalent? I am not 100% sure about the anchor elements, we want all ECGs to be delivered every time when the app launches. Symptom types should move an anchor forward.

Reply:
Good point, I think we should be fine with the setup. The new Read request elements are great!

Regarding the ECGs: We already query them with a different query so we don't need a way to reset the anchor here.

But I see a point that there should be programmatic API in place to reset an anchor if we, e.g., reset an app like we discussed today.